src/ps/proof.ts

```diff
@@ -47,7 +47,7 @@ export class PSPoKSignatureProtocol {
         }
 
         const blinding = blindings.get(idx);
-        if (blinding !== void 0) {
+        if (blinding != null) {
```

Comment:
The idea is to align behavior with other functions as all of their checks are for `== null` (at least in `/ps` dir and the code I introduced in #17).
Although, I searched over the code, and in most places, values will be checked to be strictly equal to `undefined`, so it would be great to make this behavior consistent everywhere.

I'd guess several options to choose from:
- Take a value from the map and if it's undefined (`=== undefined`), replace it with the actual blinding.
- Take a value from the map and if it's nullish (`=== null` or `=== undefined`), replace it with the actual blinding.
- Check explicitly if some value was provided in the map using `blindings.has`, and if it is, use it independently of its value.

Seeing that most of the code already checks for the `undefined`, and in places where we allow optional args they will be initialized only in case of being `undefined`, I'd say it would be better to switch to `=== undefined` checks instead.

Reply:
> And i thought you used void 0 over undefined because undefined is not a reserved word but now you are using undefined

I'm not using `undefined` in the code, I used `undefined` to distinguish `undefined` from `null` in the text. All other uses of `undefined` can be replaced too if needed.

Reply:
Ok. You can leave the existing `undefined`s in place.